pom.xml

```diff
@@ -87,25 +87,31 @@
     <app.parents>
       system:cdap-etl-batch[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-etl-realtime[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-data-pipeline[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-data-streams[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT)
     </app.parents>
+=======
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
+    <hadoop.version>2.3.0</hadoop.version>
+    <junit.version>4.11</junit.version>
+    <cdap.plugin.version>2.2.0-SNAPSHOT</cdap.plugin.version>
+>>>>>>> compile against 6.x and rename packages to io.cdap; use cdap-maven-plugin
```

Comment:
Remove this line? And like 90

Reply:
oops what is that? how could I build this locally?

Reply:
good catch - fixed and force-pushed